opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
From https://github.com/opentracing/opentracing-java/pull/7#issuecomment-173808490
> - TraceCodec does not encode or decode traces, it encodes or decodes a slender set of ids that refer to a span and a trace... and for that reason seems like a naming regression as well.

See the rationale for not calling it SpanContextCodec in the commit msg: https://github.com/opentracing/opentracing-java/commit/02d5d15bb51866afc7b8ed9d035798032c0b0848
I see your point, but by calling it the more intuitive SpanContextCodec it alludes to different SpanContexts within the one trace allowing different codec implementations.

Other suggestions?